public/app/whiteboards/reuse/reuse.html

```diff
@@ -12,9 +12,45 @@ <h2 class="modal-title">Add asset(s)</h2>
           data-search-options-user="searchOptions.user"
           data-search-options-section="searchOptions.section"
           data-search-options-type="searchOptions.type"></search>
-        <div class="col-list-container" data-infinite-scroll="getAssets()" data-infinite-scroll-ready="list.ready" data-infinite-scroll-distance="400" data-infinite-scroll-container="whiteboards-reuse-list-container">
+
+        <div class="col-list-container"
+          data-infinite-scroll="getAssets()"
+          data-infinite-scroll-ready="list.ready"
+          data-infinite-scroll-distance="400"
+          data-infinite-scroll-container="whiteboards-reuse-list-container">
+
+          <div data-ng-if="!isSearch">
+            <h3 data-ng-if="!isSearch">Pinned Assets</h3>
+            <ul>
+              <li class="list-inline col-xs-6 col-sm-4 col-md-3"
+                data-ng-repeat="asset in assets | filter:{pinned_by_me: true} | unique:'id'">
+
+                <div class="col-list-item-container">
+                  <a data-ng-click="selectAsset(asset)">
+                    <div class="col-list-item-tile">
```

Comment:
@johncrossman Could the newly created `shared/assetThumbnail.html` be subbed in here (and under "All Assets" below)? Free badges!